media/css/firefox/whatsnew/whatsnew-107-eu.scss

```diff
@@ -0,0 +1,58 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/base';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wrapper {
+    background-color: $color-violet-50;
```

Comment:
Issue (non-blocking): the background and body copy colour does pass contrast checks ok, but honestly I find it pretty hard to read still (maybe it's just me?)

<img width="1459" alt="Screenshot 2022-11-11 at 10 04 01" src="https://user-images.githubusercontent.com/400117/201316890-18bd4e47-a00d-4e5a-bd33-29625088d0ca.png">

Reply:
Not just you, I was surprised it passed color contrast too, will look into a slight adjustment (maybe lighter BG color? the larger heading font should still be ok)